docs/contributing/style-guide.md

```diff
@@ -41,4 +41,61 @@ This includes but is not limited to the following file types:
 - YAML (*.yml, *.yaml)
 - Markdown (*.md)
 - JavaScript (*.js)  
-- TypeScript (*.ts, *.tsx)
\ No newline at end of file
+- TypeScript (*.ts, *.tsx)
+
+### Error Strings
+
+Error messages should follow the standard go format. This format can be found in the [golang code review comments](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)
+however the key points are:
+
+- errors should not start with capitol letters (excluding proper nouns, acronyms, or initialism)
+- errors should not end with punctuation
+- these restrictions do not apply to logging, only the error type itself
```

Comment:
I thought we agreed on aligning the logs with errors, did not we?

Reply:
I had a longer think about it and felt maybe we should have a bigger discussion about it before we implement it everywhere. That was the intent of this PR mostly, I had planned on not merging until we all agreed. 

I'm relatively sure we all agree with error types, however when reading the go docs on why they have that style it's for logging specifically so that there isn't a random capital within a log message.

Reply:
Ok, fine with that.